signer/xpi/cose.go

```diff
@@ -5,7 +5,7 @@ import (
 	"crypto"
 	"crypto/ecdsa"
 	"crypto/rsa"
-	"crypto/sha256"
+	// "crypto/sha256"
```

Comment:
Why comment out rather than just remove?

Reply:
Yeah I forgot to finish fixing the verification code. It requires an upstream change https://github.com/mozilla-services/pkcs7/pull/56